calculator.js

```diff
@@ -1,28 +1,121 @@
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+// const exampleAdditionInput = {
+//   num1: 3,
+//   num2: 5,
+//   operation: 'add',
+// }
+// // '3 + 5 = 8'
+
+
+const addNums = function(a, b) {
+  return `${a} + ${b} = ${a + b}`;
+}
```

Comment:
Great work encapsulating the operations' functionality into functions. In the future, this is a great place to use arrow functions.

```suggestion
const addNums = (a, b) =>`${a} + ${b} = ${a + b}`;
```